src/formElementPlugins/ApplicationViewWrapper.tsx

```diff
@@ -75,7 +81,7 @@ const ApplicationViewWrapper: React.FC<ApplicationViewWrapperProps> = (props) =>
     onUpdate(currentResponse?.text)
   }, [currentResponse, isStrictPage])
 
-  const onUpdate = async (value: LooseString) => {
+  const onUpdate = async (value: LooseString, isFocusChange = false) => {
```

Comment:
I still don't quite understand about isFocusChange, I've read your explanation, but still can't quite figure it out.
To me it seems that ApplicationViewWrapper shouldn't care who is calling it.

Reply:
Yeah, I'll talk it through with you in person, it'll be easier I think.

Reply:
Okay, it looks like I can get rid of this now. Must have avoided the problem in the process of fixing/improving something else.

db57beec